src/platforms/common/configuration/options.mdx

```diff
@@ -96,6 +96,12 @@ This variable controls the total amount of breadcrumbs that should be captured.
 
 </ConfigKey>
 
+<ConfigKey name="max-cache-items" supported={[ "dotnet" ]} >
+
+The maximum number of [envelopes](https://develop.sentry.dev/sdk/envelopes/) to keep in cache. The SDKs use envelopes to send data, as events, attachments, user feedback, session, etc. to Sentry. An envelope can contain multiple items - for example, an event with a session and two attachments. Depending on the usage of the SDK, the size of an envelope can differ. When the number of envelopes in the local cache exceeds max-cache-items, the SDK deletes the oldest envelope. When deleting an envelope, we migrate sessions to the next envelope to not mess up your release health stats. The default is `30`.
```

Comment:
How about this - because it is an implementation detail, but there's value in being clear about maintaining the integrity of session stats:
```suggestion
The maximum number of [envelopes](https://develop.sentry.dev/sdk/envelopes/) to keep in cache. The SDKs use envelopes to send data, such as events, attachments, user feedback, and sessions to sentry.io. An envelope can contain multiple items, such as an event with a session and two attachments. Depending on the usage of the SDK, the size of an envelope can differ. If the number of envelopes in the local cache exceeds `max-cache-items`, the SDK deletes the oldest envelope and migrates the sessions to the next envelope to maintain the integrity of your release health stats. The default is `30`.
```